ui-ngx/src/app/shared/components/rule-chain/rule-chain-select.component.html

```diff
@@ -15,17 +15,13 @@
     limitations under the License.
 
 -->
-<mat-form-field class="tb-rule-select" subscriptSizing="dynamic">
+<mat-form-field class="tb-rule-select" subscriptSizing="dynamic" appearance="fill"
+                [ngClass]="{'cursor-pointer': !disabled}"
+                (click)="openRuleChainSelectPanel($event)">
   <mat-icon matPrefix>settings_ethernet</mat-icon>
-  <mat-select fxFlex
-              class="tb-rule-chain-select"
-              hideSingleSelectionIndicator="false"
-              [required]="required"
-              [disabled]="disabled"
-              [(ngModel)]="ruleChainId"
-              (ngModelChange)="ruleChainIdChanged()">
-    <mat-option *ngFor="let ruleChain of ruleChains$ | async" [value]="ruleChain.id.id">
-      {{ruleChain.name}}
-    </mat-option>
-  </mat-select>
+  <input matInput readonly
+         [required]="required"
+         [disabled]="disabled"
+         [value]="ruleChain?.name">
+  <mat-icon [ngClass]="{'disabled': disabled || panelOpened}" matSuffix>arrow_drop_down</mat-icon>
```

Comment:
Need to change angular binding [class](https://angular.io/guide/class-binding#binding-to-a-single-css-class).